### PR TITLE
docs: Add Linter Bear Tutorial

### PR DESCRIPTION
**Pull request by Redridge**
Fixes https://github.com/coala-analyzer/coala/issues/1491

I am aware that this will need alot of review and possibly be subject to alot of modifications.

### CODE REVIEW

File: docs/Users/Tutorials/Linter_Bears.rst [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+	(line):(column): (severity): (message)
+
+Or in `python regex <https://docs.python.org/2/library/re.html>`__
+
+::
+
+	(?P<line>\d+):(?P<column>\d+):\s(?P<severity>Error|Warning|Info):\s(?P<message>.+)
+
+Now that we found out our ``output_regex`` (we might want to test it just to be sure, I used
```
Comment (sudheesh001) [inline docs/Users/Tutorials/Linter_Bears.rst]: Maybe rephrase this to `we might want to test it just to be sure, regex101 is a great tool for such purposes`

Comment (Redridge) [inline docs/Users/Tutorials/Linter_Bears.rst]: yeah it's better

**Comment by sils1297** [comment]
ack 0aa760b

**Comment by AbdealiJK** [comment]
ack

**Comment by AbdealiJK** [comment]
ack 5a2786d

**Comment by AbdealiJK** [comment]
Moved to https://github.com/coala-analyzer/coala/pull/1530
